sys/arduino/include/Arduino.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_arduino
+ * @brief       Wrapper to keep source code compatibility for Arduino.h
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ * @{
+ */
+
+#ifndef ARDUINO_H
+#define ARDUINO_H
+
+#include "arduino.hpp"
```

Comment:
I think `#include "arduino.hpp"` should be moved inside the `#ifdef __cplusplus` block (but not inside the `extern "C"`).